### PR TITLE
Create rear.travis.yml

### PR DESCRIPTION
**Pull request by gururajrkatti**
Original .travis.yml was giving parse error. Corrected it. Also added "make install" based on the build script.

### CODE REVIEW

**Comment by seth-priya** [comment]
@gururajrkatti do we need this, I see the upstream repo at https://github.com/rear/rear/blob/master/.travis.yml already has ppc64le support?

**Comment by gururajrkatti** [comment]
Hi @seth-priya, thanks for the response and feedback. When I checked the repos, I could not see any travis build running, I tried to trigger it myself, I was seeing parse error. Hence modified it a bit. Also, based on the dockerfile in build script, I added one more step to the travis job.

**Comment by seth-priya** [comment]
Ok, that seems interesting - although I have no particular issues in merging this PR, in this case, I would actually recommend that you go back to the community and *fix* their job, especially if the issue is seen on Intel as well? The install step maybe unnecessary in travis.yml , but again no harm in having it in either

File: travis-ymls/rear.travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : rear
+# Source Repo         : https://github.com/gururajrkatti/rear
```
Comment (seth-priya) [inline travis-ymls/rear.travis.yml:4]: Please mention the link to the original repo. instead of the forked one.

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by gururajrkatti** [comment]
@seth-priya, community has accepted this change

**Comment by seth-priya** [comment]
Thanks @gururajrkatti for the update and for following through on this! In that case I don't think we need  it here

**Pull request closed without merging**